2023/src/templates/speaker.tsx

```diff
@@ -112,19 +112,34 @@ export default function Speaker(props: Props) {
       />
       <ResponsiveBox>
         <Breadcrumb path={[{ label: t("speakers"), to: "/speakers" }, title]} />
-        {speakers.map((speaker, i) => (
-          <SpeakerBox key={speaker.uuid}>
-            <Avatar image={avatars[i]} alt={speaker.name} loading="eager" />
+        {speakers.map((speaker, i) => {
+          const socialLinks = [
+            {account: speaker.github, site: 'github'},
+            {account: speaker.mastodon, site: 'mastodon'},
+            {account: speaker.twitter, site: 'twitter'}
+          ].filter(item => item.account).map((item) => {
+            let listItem =  ''
+            if (item.site == 'github') listItem = `- Github: [${item.account}](https://github.com/${item.account})\n`
```

Comment:
It would be nice if you could use `npm i @styled-icons/remix-line` to add a list of icons after the markdown code.